packages/landing/src/components/debug.tsx

```diff
@@ -282,7 +282,9 @@ export class Debug extends Component<{ map: maplibregl.Map }, DebugState> {
       return;
     }
 
-    if (currentTerrain?.source === sourceId) return;
+    const target = getTerrainForSource(sourceId, Config.map.tileMatrix.projection);
+    // no changes
+    if ((currentTerrain?.source === sourceId, currentTerrain?.exaggeration === target.exaggeration)) return;
```

Comment:
That's a new conditional check format that I never seen before. I suppose it works same as `&&`?

Reply:
very weird, wonder how that happened!